server/shared/database/migrations/20210119091445-add-resolved-and-edited-at-column-to-comment.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const TABLE_NAME = 'comment';
+
+const getColumns = ({ BOOLEAN, DATE }) => [
+  { name: 'resolved', type: BOOLEAN, defaultValue: false },
+  { name: 'edited_at', type: DATE, defaultValue: null }
+];
+
+exports.up = async (qi, Sequelize) => {
+  const columns = getColumns(Sequelize);
+  const transaction = await qi.sequelize.transaction();
+  await Promise.all(columns.map(({ name, ...options }) => {
```

Comment:
Hmm, I think It will be better to decouple this in 2 separate migrations to avoid using `bluebird` injection (to handle promises in series), and also to avoid using transaction. Also it's more readable (code, file naming..)